keycloak-theme/pom.xml

```
@@ -237,7 +237,7 @@
                             <goal>npm</goal>
                         </goals>
                         <configuration>
-                            <arguments>run build --workspace=admin-ui</arguments>
+                            <arguments>run build --workspace=keycloak-js --workspace=admin-ui</arguments>
```

Why not build all workspaces here? like this:
```suggestion
                            <arguments>run build --workspaces</arguments>
```

---

Since we are possibly adding more in the future I decided to be explicit here for now. Likely this will be replaced with some toolt that can track dependencies for us (such as [wireit](https://github.com/google/wireit)), so it's temporary either way.